MAUI/Popup/popup-positioning.md

```diff
@@ -0,0 +1,266 @@
+<td>Shows the `SfPopup` at the specified X and y positions.</td></tr>
+<tr>
+<td>SfPopup.ShowRelativeToView(View, RelativePosition)</td>
+<td>Shows the `SfPopup` at the position relative to the specified view.</td></tr>
+<tr>
+<td>SfPopup.ShowRelativeToView(View, RelativePosition,x-position,y-position)</td>
+<td>Shows the `SfPopup` at an absolute x, y coordinate from the relative position of the specified view.</td></tr>
+<tr>
+<td>SfPopup.IsFullScreen</td>
+<td>Shows the `SfPopup` in full width and height of the screen.</td></tr>
+<tr>
+<td>SfPopu.Show(bool)</td>
+<td>Similar as `SfPopup.IsFullScreen` property.</td></tr>
+</table>
+
+## Center positioning
+
+`SfPopup` can be shown at the center by using the following options.
+
+ * IsOpen property
+ * SfPopupLayout.Show
+
+To open the `SfPopup`, use the `SfPopup.IsOpen` property as in the following code sample.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupMauiPositioning.MainPage">
+  <ContentPage.Content>
+    <StackLayout Padding="20">
+      <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+              VerticalOptions="Start" HorizontalOptions="Center" 
+              Clicked="ClickToShowPopup_Clicked" />
+      <sfPopup:SfPopup x:Name="sfPopup">
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the center of the view.
+	sfPopup.IsOpen = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+To open the SfPopup, use the SfPopup.Show method as in the following code sample.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupMauiPositioning.MainPage">
+  <ContentPage.Content>
+    <StackLayout Padding="20">
+      <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+              VerticalOptions="Start" HorizontalOptions="Center" 
+              Clicked="ClickToShowPopup_Clicked" />
+      <sfPopup:SfPopup x:Name="sfPopup">
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the center of the view.
+	sfPopup.Show();
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Popup with center positioning](Images/popup-positioning/maui-popup-with-center-positioning.png)

```

Comment:
![Displaying a .NET MAUI Popup at the center of the screen.]
Include .NET MAUI in all image alt text